test/driver.js

```diff
@@ -604,6 +606,16 @@ var Driver = (function DriverClosure() {
                 viewport,
                 renderInteractiveForms: renderForms,
               };
+              if (renderPrint) {
+                const annotationStorage = task.annotationStorage;
+                const docAnnotationStorage = task.pdfDoc.annotationStorage;
+                for (const [key, value] of Object.entries(annotationStorage)) {
+                  docAnnotationStorage.setValue(key, value);
+                }
+                renderContext.annotationStorage = docAnnotationStorage;
+                renderContext.intent = "print";
+              }
```

Comment:
Would there also, generally, be a good idea to be able to test the "print" intent code-paths without the need for an `annotationStorage` to be present?

If so, it may perhaps make sense to do something like the below!?
```suggestion
              if (renderPrint) {
                const annotationStorage = task.annotationStorage;
                if (annotationStorage) {
                  const docAnnotationStorage = task.pdfDoc.annotationStorage;
                  for (const [key, value] of Object.entries(annotationStorage)) {
                    docAnnotationStorage.setValue(key, value);
                  }
                  renderContext.annotationStorage = docAnnotationStorage;
                }
                renderContext.intent = "print";
              }
```

Reply:
I thought about this, but normally the print intent would be equal to the display intent, so it's not useful in that case. Only in the case of AcroForms the print rendering differs from the display rendering because the operator list is altered according to the form data present in the annotation storage. Therefore, for now I thought it would be best to let printing and annotation storage go hand-in-hand, and if we find a use case for leaving out the annotation storage that can simply be changed later on.

Reply:
There's a couple of cases that come to mind, for example:
 - Any "regular" Annotation may be printable, but not viewable, hence the intent may even matter for simple documents.
 - Some optional content groups can be marked as printing-only, although we don't yet support that.

Generally speaking, there's probably a lot more things that can be printing-only and where being able to test with just `"print": true` makes sense.
Hence I'd still recommend making these changes here, since unless I'm missing something it ought to be almost trivial and avoids churn later on :-)

Reply:
Ah, yes. I forgot about the printable-only annotations and didn't know about the printable-only optional content groups. In that case it's fair to not require the annotation storage, so I'll make the change you suggested. Thanks!